yolapy/models/user.py

```diff
@@ -0,0 +1,54 @@
+from yoconfig import get_config
+from yolapy.services import Yola
+
+
+class User(object):
+
+    """Yola User."""
+
+    def __init__(self, active=False, deleted=None, email=None, id=None,
+                 name=None, partner_id=None, preferences=None,
+                 signup_date=None, surname=None):
```

Comment:
I would prefer that the model not dictate how it must be used.

A slightly ridiculous counter example (but IMO valid), lets say I want to: 

1. Create a yola user
1. Pass it to the "get name" routine
1. Pass it to the "attach email" routine
1. Save

Indeed, the service will throw an error if `email`, `name` or `surname` is missing (although, names can be empty strings) but the init does not seem like the best place to capture those business rules.  Maybe a "validate" method may one day be nice but ATM I'm okay with relegating validation to the service and letting it respond 400 + reason when the data is incomplete.